redis-namespace.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |s|
   s.files            += Dir.glob("test/**/*")
   s.files            += Dir.glob("spec/**/*")
 
-  s.required_ruby_version = '~> 2.4'
+  s.required_ruby_version = ['>= 2.4', '< 3.1']
```

Comment:
Let's relax this to just the `>= 2.4` lower bound. We'll expect to remain compatible with future Ruby versions rather than forbidding it.

Reply:
Done.